plugins/modules/cloud/misc/terraform.py

```diff
@@ -80,9 +80,15 @@
     aliases: [ 'variables_file' ]
   variables:
     description:
-      - A group of key-values to override template variables or those in
-        variables files.
+      - A group of key-values to override template variables or those in variables files.
+      - Support complex variable structures to reflect terraform variable syntax.
+      - Terraform Objects are mapped to Ansible Dicts.
```

Comment:
```suggestion
      - Terraform objects are mapped to Ansible dictionaries.
```
Similar changes for the next paragraphs.